Added number-pane in score reporting

SirDroun opened the pull request:
Simplify for small devices to quickly find patrol
when reporting scores. User can just start
typing the unique id of patrol to filter (and
select first match) of patrol.

Commit:
Added number-pane in score reporting

Simplify for small devices to quickly find patrol
when reporting scores. User can just start
typing the unique id of patrol to filter (and
select first match) of patrol.

erydberg commented:
Hej, jag ser att din pull request ligger en commit före och en commit efter min master-branch. Kan du ta ner min senaste master in i din branch först så vi inte får någon konflikt? Jag är inte så bra på pull requests ännu...

erydberg commented:
@SirDroun Jag testade att bygga med dina ändringar och installerade det nya paketet på min utv-server. Själva funktionen är bra men i mina tester hamnar alla knapparna över dropdownlistan för patruller och det blir grötigt, åtminstone i min mobilemulator.

SirDroun commented:
He, jo, det kan bli så på en liten skärm. Lägg mobilen ned! Det var en poäng med att lägga den vid sidan av pull-down, snabb-access och feedback, men vi får väl ändra lite i stylsheet annars. Hur tänker du att man vill ha den?
Har du pushat något mer nyligen? Jag trackade din master innan, så den borde ligga på det. Men jag fixar en ny.

erydberg commented:
Jag tror inte jag ändrat något som krockade med din branch men det stod att
din pull request var en commit efter master (och en före också förstås).

Ett alternativ är att ha den över eller under droplistan fast dold och en
enkel "fälla ner" (toggle visa dölj) snabbinmatningen-länk som gör att
funkar bra med en stående mobil.

Mvh
Erik

Den 21 september 2016 22:13 skrev Emil Öberg notifications@github.com:

> He, jo, det kan bli så på en liten skärm. Lägg mobilen ned! Det var en
> poäng med att lägga den vid sidan av pull-down, snabb-access och feedback,
> men vi får väl ändra lite i stylsheet annars. Hur tänker du att man vill ha
> den?
> Har du pushat något mer nyligen? Jag trackade din master innan, så den
> borde ligga på det. Men jag fixar en ny.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> https://github.com/erydberg/gokopen/pull/38#issuecomment-248729083, or mute
> the thread
> https://github.com/notifications/unsubscribe-auth/AHbpBK7sbeIIUx9qIlSkgnLmK6lNlzsGks5qsY_cgaJpZM4KCeRH
> .

SirDroun commented:
Då är det nog bättre att alltid ha den nedanför. Eller att rekommendera att man har mobilen liggandes.
Jag flyttar ned den under så får vi se hur det blir. Ny commit kommer imorgon.

SirDroun commented:
Fråga förresten. Ska startkorten se ut så? eller är det jag som får konstig Christmas-tree alignment på dem? De borde kunna bli kompaktare och mer lättlästa? Jag kan försöka lägga på en ny css på dem så kan vi se.

erydberg commented:
Jag tycker startkorten blir ok hos mig och vid utskrift.

Den 21 september 2016 23:12 skrev Emil Öberg notifications@github.com:

> Fråga förresten. Ska startkorten se ut så? eller är det jag som får
> konstig Christmas-tree alignment på dem? De borde kunna bli kompaktare och
> mer lättlästa? Jag kan försöka lägga på en ny css på dem så kan vi se.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> https://github.com/erydberg/gokopen/pull/38#issuecomment-248744622, or mute
> the thread
> https://github.com/notifications/unsubscribe-auth/AHbpBFjp6_-lFt9JygKBPddmOrNRU2Nuks5qsZ3CgaJpZM4KCeRH
> .

Commit:
Stop filter pane overlap

Filter pane on tiny screen overlaped pulldown instead of
sliding down below it.

erydberg commented:
Hej

Nu på söndag är det terminens sista ordinarie möte för äventyrarna. Vi
startar 13 och håller på till klockan 15 och är i Hällingsjö. Vi kommer
vara ute en del så varma kläder efter väder. Ta med kniv.

Söndagen den 15 december har vi julavslutning på Hålemyst. Mer information
kommer.

Scouthälsningar
Mattias och Erik
